From: Modestas Vainius <modax@debian.org>
Subject: Do not check for clamz at build-time
Forwarded: not-needed
Origin: vendor
Last-Update: 2013-01-17

Clamz is a runtime dependency. There is no need to check for it at build time.
Index: amarok/CMakeLists.txt
===================================================================
--- amarok.orig/CMakeLists.txt	2014-09-24 12:54:01.740634134 +0200
+++ amarok/CMakeLists.txt	2014-09-24 12:54:01.740634134 +0200
@@ -216,9 +216,6 @@
     macro_log_feature( GLIB2_FOUND "glib2" "Required by libgpod and mp3tunes" "http://www.gtk.org" FALSE "2.x" "")
     endif( WITH_IPOD OR WITH_MP3Tunes )
 
-    find_program( CLAMZ_FOUND clamz PATH )
-    macro_log_feature( CLAMZ_FOUND "clamz" "Optional requirement to download songs from the Amazon MP3 store. Highly recommended on Linux, as the official downloader from Amazon is quite broken on many systems." "https://code.google.com/p/clamz/" FALSE )
-
     include_directories( ${KDE4_INCLUDES} )
 
     if( KDE4_BUILD_TESTS AND NOT WIN32 )
